gwcli/ceph.py

```diff
@@ -288,11 +288,20 @@ class RadosPool(UINode):
     def __init__(self, parent, pool_name, pool_md):
         UINode.__init__(self, pool_name, parent)
         self.pool_md = pool_md
+        k = 'N'
+        m = 'N'
+        if pool_md['type'] == 3:
+            _result = shellcommand(
+                "ceph osd erasure-code-profile get {} --format json".
+                format(pool_md['erasure_code_profile']))
```

Comment:
I took a deeper look and I don't think `shellcommand` is needed at all.  The EC profile information is already there in `CephPools.populate` in the `osd dump` reply.  Just refactor it a bit and pass `osd_dump['erasure_code_profiles']` to `RadosPool` constructor.

Reply:
Done.